StyleCop.Analyzers/StyleCop.Analyzers/DocumentationRules/SA1619GenericTypeParametersMustBeDocumentedPartialClass.cs

```diff
@@ -67,6 +67,7 @@
     /// tags on this part of the class.</para>
     /// </remarks>
     [DiagnosticAnalyzer(LanguageNames.CSharp)]
+    [NoCodeFix("Cannot generate documentation")]
```

Comment:
:exclamation: I believe this one can be generated, at least as an element containing no text. The result would then produce warning SA1622, but at least the user wouldn't have to type the XML portion manually.

```csharp
/// <typeparam name="S"></typeparam>
```